docs/posts/ibis-v8-release/index.qmd

```diff
@@ -0,0 +1,269 @@
+---
+title: "Ibis v8: streaming and more!"
+author: "Ibis team"
+date: "2024-02-05"
+categories:
+    - release
+    - blog
+---
+
+## Overview
+
+Ibis v8 marks the first release of data streaming backends in Ibis! We're
+excited to begin bridging the gap between batch and streaming systems, starting
+with [Apache Flink](https://flink.apache.org/) and
+[RisingWave](https://risingwave.com) backends in this release. We've also added
+a new batch backend with [Exasol](https://exasol.com), bringing the total number
+of backends Ibis supports to 20.
+
+[Most geospatial operations are now supported in the DuckDB
+backend](#geospatial-operations-in-duckdb), making Ibis a great local option for
+geospatial analytics.
+
+### What is data streaming?
+
+Data streaming is a method of processing data continuously as it arrives rather
```

Comment:
a key component of streaming is unbounded data processing

Reply:
technically you can still do stream processing on historical data using watermark, so the key differentiation between streaming and batch is unbounded processing with time semantics.